### ISSUE TITLE
Request: any chance of adding Large Biomes searching to cubiomes? :)

### ISSUE THREAD

**Issue by Killstepz** [Cubitect/cubiomes-viewer#48]
i know i alrdy asked for this a while ago but i tought id bring it up again :)
This would save tons of hours of searching for the perfect seed

**Comment by Boobies** [comment]
By all means, spam all the issue trackers on GitHub, they are your playground. Smiley face, wink wink. Is this useful or do you think you are pestering people? I will leave it up to you to think about it.

**Comment by Cubitect** [comment]
I have closed your other issue as it essentially contained two unrelated requests for
A) an option for adding shadows of all seeds in the list (which is possible since v1.7), and
B) support for large biomes, which this issue is just a copy of.

Regarding large biomes, I have already made some efforts to implement this before, but I have run into some difficulties. I will leave this open, but it may be a while before I get back to this.
